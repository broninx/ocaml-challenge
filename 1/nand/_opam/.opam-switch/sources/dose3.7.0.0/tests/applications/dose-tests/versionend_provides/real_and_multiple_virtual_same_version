Package: main
Version: 1
Architecture: amd64
Depends: d, b, c
Description:
 forces to install a real package a(=1), and two packages providing a(=1) 

Package: a
Version: 2
Architecture: amd64
Provides: d

Package: b
Version: 1
Architecture: amd64
Provides: a (=1)

Package: c
Version: 42
Architecture: amd64
Provides: a (=1)
