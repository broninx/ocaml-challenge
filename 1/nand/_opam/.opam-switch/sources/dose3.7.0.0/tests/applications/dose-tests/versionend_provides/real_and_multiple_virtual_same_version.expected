output-version: 1.2
report:
 -
  package: main
  version: 1
  architecture: amd64
  status: ok
  
 
background-packages: 3
foreground-packages: 1
total-packages: 4
broken-packages: 0
